src/ts/commands.ts

```diff
@@ -265,6 +266,23 @@ export const applyAutoFixableSuggestionsCommand = <TMatch extends IMatch>(
   return maybeApplySuggestions(suggestionsToApply, state, dispatch);
 };
 
+/**
+ * Ignore a match, removing it from the plugin state.
+ * Returns true if the match was found, false if not.
+ */
+export const ignoreMatchCommand = (id: string) => <TMatch extends IMatch>(
+  getState: GetState<TMatch>
+) => (
+  state: EditorState,
+  dispatch?: (tr: Transaction<any>) => void
+): boolean => {
+  const match = selectMatchByMatchId(getState(state), id);
+  if (match && dispatch) {
+    dispatch(state.tr.setMeta(PROSEMIRROR_TYPERIGHTER_ACTION, removeMatch(id)));
+  }
+  return !!match;
```

Comment:
Why bother returning?  If it's idempotent, we don't care.

Reply:
It'll tell the consumer whether it's found the match or not, as the function comment says ... having said that, nothing uses that API downstream, so arguably redundant.

Reply:
For slightly more context – prosemirror commands have an API that makes dispatch optional, so we can run commands to see if something is possible without doing it – this would enable that behaviour to fit that convention. See 'Commands' in https://prosemirror.net/docs/guide/ for a description, this behaviour is what turns the menu icons in Composer on and off, for example.